awx/main/migrations/0147_unifiedjob_receptor_unit_id.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.16 on 2021-06-11 04:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0146_add_insights_inventory'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='unifiedjob',
+            name='receptor_unit_id',
```

Comment:
I have this lurking fear that receptor will get renamed in the future. Maybe we can call it `work_unit_id` ? (I'll let you make the call, the rename wound is fresh)